RUNTIME ERROR: foo
	error.05.jsonnet:17:21-31	object <ErrObj>
	error.05.jsonnet:18:25-30	object <anonymous>
	During manifestation	
